### ISSUE TITLE
Vulnerability - What is stopping the buyer listing the item straight onto the steam community market as soon as it is received? Seller will not have time to scan his inventory and escrow will be in favor of the buyer 

### ISSUE THREAD

**Issue by JohnnyBoyyy** [sugarskins/csgo-steam-eth-trade#21]
Vulnerability - What is stopping the buyer listing the item straight onto the steam community market as soon as it is received? Seller will not have time to scan his inventory and escrow will be in favor of the buyer

**Comment by danoctavian** [comment]
@JohnnyBoyyy you are correct. When i initially started the project i thought the trade lock doesn't allow you to put the item on the market.

Later on i learned that's not the case so you can't trade it but you can still put in on the market instantly, so the exploit you are explaining renders this whole project useless.
